src/server/tx_base.h

```diff
@@ -59,6 +59,10 @@ struct KeyIndex {
 struct DbContext {
   DbIndex db_index = 0;
   uint64_t time_now_ms = 0;
+
+  DbContext() = default;
+  DbContext(DbIndex dbid, uint64_t now_ms) : db_index(dbid), time_now_ms(now_ms) {
+  }
```

Comment:
fun fact, all the code you wrote existed implicitly 🙂

Reply:
Honestly initialization in C++ is wild west :rofl: For example some people prefer this because it's explicit:

```
  struct Foo{ int i; int k; };
  Foo foo{.i = 3, .k = 4};
  
  whereas others 
  
  Foo foo(3, 4);
```

Argument for the former is that it's explicit in what members you initialize. Same applies for `default constructors`. 
Is it worth to be explicit with what is `implicitly defined by the compiler` or not? If you `delete` something most certainly because it's hard to remember all the rules but again there are exceptions in few cases that are very apparent

Maybe the google style guide has a clause for this :shrug:

Reply:
Roman uses only `{a, b}` which also implcitly existed before, only the casting rules got relaxed

Reply:
struct initialization does not compile for older compilers so I am used to be explicit to be on the safe side.